src/app/components/select-field/index.html

```diff
@@ -63,6 +63,13 @@
     >
     Specifies the selection mode that determines whether users can select one item or multiple items in the select field component's modal. The valid options are <sky-code>single</sky-code>, which displays a text input and allows users to select one item, and <sky-code>multiple</sky-code>, which displays a button and allows users to select multiple items.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="showAddNewRecordButton"
+      isOptional="true"
+      defaultValue="false"
+    >
+    Indicates whether to display a button for users to add new items directly from the select field modal.
```

Comment:
This will only show the New button; it doesn't automatically add items to the list (the consumer must provide that logic themselves). Is it worth pointing out that distinction?

Reply:
It seems worth pointing out the distinction to me. What do you think of adding a second sentence like: "Indicates whether to display a button for users to add new items directly from the select field modal. You must provide the logic for adding items."?